package-lock.json

```diff
@@ -295,9 +295,26 @@
         "@angular/elements": "^13.2.3"
       }
     },
+    "dist/valtimo/components/node_modules/@ng-bootstrap/ng-bootstrap": {
+      "version": "12.1.2",
```

Comment:
Is it okay that the package-lock still contains the wrong version?

Reply:
Goeie fix ik ff